frontend/components/droplets/lessons/lesson-renderer.tsx

```
@@ -59,19 +61,47 @@ interface HighlightResponseItem {
   };
 }
 
+function renderTextWithStyles(
+  text: string,
+  styles?: Record<string, any>,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)